apps/core/scenes/results/Results.js

```diff
@@ -130,19 +131,26 @@ class Results extends React.Component<Props> {
     };
     return (
       <SafeAreaView style={styles.container}>
-        <SearchParamsSummary
-          tripType={tripType}
-          departure={{
-            city: travelFromName,
-            localizedDate: getFormattedDate([dateFrom, dateTo]),
-          }}
-          arrival={{
-            city: travelToName,
-            localizedDate: returnDateFrom
-              ? getFormattedDate([returnDateFrom, returnDateTo])
-              : '',
-          }}
-        />
+        {Platform.OS === 'mobile ' ? (
```

Comment:
Extra space in `mobile` is causing this.
<img width="391" alt="Screenshot 2019-04-16 at 11 40 18" src="https://user-images.githubusercontent.com/2660330/56199150-a1c72d00-603c-11e9-9d0f-761c43d3c7f2.png">